commons-exo-extension/src/main/webapp/WEB-INF/gatein-resources.xml

```diff
@@ -30,4 +30,31 @@
       <module>eXoVueI18n</module>
     </depends>
   </module>
+  <module>
+    <name>WYSIWYGPluginsExtensions</name>
+    <load-group>NotesEditorGRP</load-group>
+    <script>
+      <minify>false</minify>
+      <path>/js/WYSIWYGPluginsExtensions.bundle.js</path>
+    </script>
+    <depends>
+      <module>vue</module>
+    </depends>
+    <depends>
+      <module>eXoVueI18n</module>
+    </depends>
+    <depends>
+      <module>vuetify</module>
+    </depends>
+    <depends>
+      <module>jquery</module>
+      <as>$</as>
+    </depends>
+    <depends>
+      <module>extensionRegistry</module>
+    </depends>
+    <depends>
+      <module>commonVueComponents</module>
+    </depends>
```

Comment:
Does this extension need all those dependencies ?

Reply:
Updated !